Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.6.2-alpine
+
+RUN apk add --update git && apk add --update make && rm -rf /var/cache/apk/*
+
+ADD . /go/src/github.com/ernestio/network-creator-aws-connector
```

Comment:
This is not compatible with different organisation names, so in the case we work with a modified component stored on github.com/r3labs/network-creator-aws-connector is not going to work.

Any idea on how to make this work with multi-org approach?